alpha/lib/model/assetPeer.php

```diff
@@ -590,6 +604,42 @@ public static function retrieveHighestBitrateByEntryId($entryId, $tag = null, $e
 	}
 
 	/**
+	 * if $retrieveHighestBitrate is set to true we will retrieve true if flavotAsset bitrate is higher then ret bitrate
+	 * if $retrieveHighestBitrate is set to false we will retrieve true if flavotAsset bitrate is lower then ret bitrate
+	 *
+	 */
+	public static function replaceSavedFlavorWithBetterBitrateFlavor($flavorAsset, $ret, $retrieveHighestBitrate)
```

Comment:
1. Better rename replaceSavedFlavorWithBetterBitrateFlavor to replaceFlavorWithBetterBitrateFlavor 
2. What is $ret? is it a type of flavorAsset? please rename to a more meaningful name 
3. If you assume that $flavorAsset and$ret are type of flavorAsset , better state the type in the function definition to protect against null input
4. Is this function really needs to be public?